package.json

```diff
@@ -55,6 +57,7 @@
     "@types/luxon": "2.3.2",
     "@types/node": "16.11.33",
     "@types/sinon": "10.0.11",
+    "@typescript-eslint/eslint-plugin": "^5.26.0",
```

Comment:
oh dang. Was this required? `eslint-config` should be pulling this in automatically :(

Reply:
oh, I was getting an error running locally. I can remove and try again